test/socket_client_test.exs

```diff
@@ -69,12 +69,11 @@ defmodule Phoenix.Channels.GenSocketClientTest do
     assert {:ok, 5} == TestSocket.push(conn.socket, "channel:2", "channel_2_event")
     assert {:ok, 6} == TestSocket.push(conn.socket, "channel:1", "yet_another_event")
 
-    # leave and rejoin the channel, and verify that counter has been reset
+    # leave and rejoin the channel
```

Comment:
This comment is somewhat pointless at this point. Maybe describe _why_ you are leaving and rejoining?

Reply:
Yeah, this test is actually pointless, as we're no longer resetting references.

Reply:
Well, I guess I could still keep the part before join/leave to prove that references are increasing.